src/execution/operator/persistent/buffered_csv_reader.cpp

```diff
@@ -1751,6 +1771,10 @@ void BufferedCSVReader::Flush(DataChunk &insert_chunk) {
 				success = VectorOperations::TryCast(parse_chunk.data[col_idx], insert_chunk.data[col_idx],
 				                                    parse_chunk.size(), &error_message);
 			}
+			if (options.ignore_errors == true && !success) {
```

Comment:
nit: `options.ignore_errors`

Perhaps we can slightly rewrite this to continue on success instead:

```cpp
if (success) {
    continue;
}
if (options.ignore_errors) {
    conversion_error_ignored = true;
    continue;
}
// throw exception